### PR TITLE
Template update

### LINKED ISSUE #4: Update pipeline template
### Description of feature

Update to the latest nf-core pipeline template

### LINKED ISSUE #7: Add CI tests on the farm
### Description of feature

And a "cleanup" profile

### PR DESCRIPTION
**Pull request by muffato**
Closes #4 
I've updated the pipeline template from 2.5 to 2.8.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/ensemblgenedownload/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md
[the remark this answers is not in the document]
Comment (priyanka-surana) [reply docs/usage.md:1]: This did not work for me during testing. It kept dying on `geneset version`. Also `geneset_version` is refrenced here as both `yyyy-mm` and `yyyy_mm`. Can you clean it up?

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
Mostly need more clarity for input parameters for ToL and non-ToL assemblies. The other comments are similar to `insdcdownload`.

File: .nf-core.yml
[the remark this answers is not in the document]
Comment (muffato) [reply .nf-core.yml:1]: I added the code of conduct and could remove this line. The rest needs to stay

**Review by priyanka-surana — APPROVED** [review]
LGTM 👍🏼 Thanks for making the changes.

**Pull request merged**